src/snowflake/cli/plugins/git/commands.py

```diff
@@ -13,7 +17,22 @@
 )
 log = logging.getLogger(__name__)
 
+
+def _repo_path_argument_callback(path):
+    if not is_stage_path(path):
+        raise ClickException(
+            f"REPOSITORY_PATH should be a path to git repository stage with scope provided."
+            " For example: @my_repo/branches/main"
```

Comment:
`@my_repo/branches/main` - is it still a valid example (no `/` at the end)? If it isn't, then please fix it here and in other places. And maybe `RepoPathArgument` help should say when to use `/` at the end and when not.